integration_tests/e2e/cellConversion/editing.cy.ts

```diff
@@ -132,6 +132,31 @@ context('Cell conversion', () => {
         Page.verifyOnPage(CellConversionConfirmPage)
         cy.get('.govuk-summary-list__value').eq(1).contains('First night centre / Induction')
       })
+
+      it('can uncheck the used for types and leave XXXXX ', () => {
+        const usedForPage = Page.verifyOnPage(CellConversionUsedForPage)
+        usedForPage.usedForCheckboxItem('CLOSE_SUPERVISION_CENTRE').click()
+        usedForPage.usedForCheckboxItem('SUB_MISUSE_DRUG_RECOVERY').click()
+        usedForPage.usedForCheckboxItem('FIRST_NIGHT_CENTRE').click()
+        usedForPage.continueButton().click()
+        const confirmPage = Page.verifyOnPage(CellConversionConfirmPage)
+        cy.get('.govuk-summary-list__value').eq(1).contains('First night centre / Induction')
+        confirmPage.backLink().click()
+        const cellConversionSetCellCapacityPage = Page.verifyOnPage(CellConversionSetCellCapacityPage)
+        cellConversionSetCellCapacityPage.backLink().click()
+        const setCellTypePage = Page.verifyOnPage(CellConversionSetCellTypePage)
+        setCellTypePage.cellTypeCheckboxItem('ACCESSIBLE_CELL').click()
+        setCellTypePage.cellTypeCheckboxItem('BIOHAZARD_DIRTY_PROTEST').click()
+        usedForPage.continueButton().click()
+        Page.verifyOnPage(CellConversionSetCellTypePage)
+        setCellTypePage.backLink().click()
+        setCellTypePage.backLink().click()
+        usedForPage.usedForCheckboxItem('CLOSE_SUPERVISION_CENTRE').uncheck()
+        usedForPage.usedForCheckboxItem('SUB_MISUSE_DRUG_RECOVERY').uncheck()
+        usedForPage.usedForCheckboxItem('FIRST_NIGHT_CENTRE').uncheck()
+        usedForPage.continueButton().click()
+        Page.verifyOnPage(CellConversionUsedForPage)
+      })
```

Comment:
This test covers editing the value after you have got to the confirmation page. The scenario described in the ticket is different. We need a test in `integration_tests/e2e/cellConversion/index.cy.ts` that follows the steps in the video.